packages/app/next.config.js

```diff
@@ -9,6 +9,7 @@ const withNextra = require('nextra')({
 });
 
 module.exports = {
+  basePath: process.env.HYPERDX_BASE_PATH || '',
```

Comment:
Let's call it `NEXT_PUBLIC_BASEPATH`. We also need to verify if this works on production build